rmw_dds_common/include/rmw_dds_common/graph_cache.hpp

```diff
@@ -474,9 +472,13 @@ class GraphCache
   /// Copy the names and namespaces of the discovered nodes.
   /**
    * \param[inout] node_names A zero initialized string array, where the node names will be copied.
+   *   Must not be `nullptr`.
    * \param[inout] node_namespaces A zero initialized string array, where the node namespaces
    *   will be copied. Each item in this array corresponds to an item in the same position of
-   *   node_names array.
+   *   node_names array. Must not be `nullptr`.
+   * \param[inout] security_contexts A zero initialized string array, where the context name of the node
```

Comment:
@ivanpauno nit: 
```suggestion
   * \param[inout] security_contexts A zero initialized string array, where the security context name of the node
```

so we're consistent about what's a security context and what's a context (despite being used interchangeably now).

Reply:
Done in https://github.com/ros2/rmw_dds_common/pull/4/commits/b7f8ce4e6986b96a49838a562510e3266d3a4222.